caffe2/python/modeling/parameter_info.py

```diff
@@ -15,7 +15,7 @@ class ParameterType(object):
 class ParameterInfo(object):
     def __init__(
             self, param_id, param, key=None, shape=None, length=None,
-            grad=None):
+            grad=None, blob_copy=None):
```

Comment:
I already added that in the body of the method. Would you prefer it prefixing the function? Inline with the signature?